finders.go

```diff
@@ -22,22 +21,12 @@ func FindDirect(cfg Config) []string {
 		cfg.Logger.Debugf("unable to expand path: %s", cfg.File)
 		file = cfg.File
 	}
-	if fileExists(file) {
-		return []string{file}
-	}
-	return nil
+	return []string{file}
 }
 
 // FindConfigYamlInCwd looks for ./config.yaml -- NOTE: this is not part of the default behavior
-func FindConfigYamlInCwd(cfg Config) []string {
-	// check if config.yaml exists in the current directory
-	f := "./config.yaml"
-	if fileExists(f) {
-		cfg.Logger.Warnf("DEPRECATED: %s as a configuration file is deprecated and will be removed as an option in v1.0.0, please rename to .syft.yaml", f)
-		return []string{f}
-	}
-
-	return nil
+func FindConfigYamlInCwd(_ Config) []string {
```

Comment:
should we even include this finder at all in this lib?

Reply:
I think it's a possibly valid option someone would want -- to load `./config.yaml`, which is why I've included it here (even though it's a "bug"). It's nice to be already tested consistently, this way.